README.md

```diff
@@ -57,70 +171,195 @@ Second XBlock instance:
         <tip display="yes">Great!</tip>
         <tip reject="maybenot">Ah, damn.</tip>
         <tip reject="understand"><html><div id="test-custom-html">Really?</div></html></tip>
-    </quizz>
+    </mcq>
 
-    <quizz name="quizz_1_2" type="rating" low="Not good at all" high="Extremely good">
-        <question>How much do you rate this quizz?</question>
+    <mcq name="mcq_1_2" type="rating" low="Not good at all" high="Extremely good" weight="5">
+        <question>How much do you rate this MCQ?</question>
         <choice value="notwant">I don't want to rate it</choice>
 
         <tip display="4,5">I love good grades.</tip>
         <tip reject="1,2,3">Will do better next time...</tip>
         <tip reject="notwant">Your loss!</tip>
-    </quizz>
+    </mcq>
+</mentoring>
+```
+
+Before attempting to answer the questions:
+
+![MCQ Initial](https://raw.githubusercontent.com/edx-solutions/xblock-mentoring/1fb5e3ece6f34b6cf33c956a4fba1d7cbb7349a2/doc/img/mcq-1.png)
+
+While attempting to complete the questions:
+
+![MCQ Attempting](https://raw.githubusercontent.com/edx-solutions/xblock-mentoring/1fb5e3ece6f34b6cf33c956a4fba1d7cbb7349a2/doc/img/mcq-2.png)
+
+After successfully completing the questions:
+
+![MCQ Success](https://raw.githubusercontent.com/edx-solutions/xblock-mentoring/1fb5e3ece6f34b6cf33c956a4fba1d7cbb7349a2/doc/img/mcq-3.png)
+
+### Self-assessment MRQs
 
-    <message type="completed">
-        All is good now...
-        <html><p>Congratulations!</p></html>
-    </message>
+Multiple response questions are set up similarly to MCQs. The answers
+are rendered as checkboxes. Unlike MCQs where only a single answer can
+be selected, MRQs allow multiple answers to be selected at the same
+time.
+
+The `<mrq>` element supports these attributes:
```

Comment:
In fact, the README is so saturated XML stuff, I'm going to stop looking at it for now, and ask that you ping me once you've given it a run over so I can look at it again.